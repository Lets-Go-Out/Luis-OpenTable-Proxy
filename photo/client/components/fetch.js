const $ = require("jquery");
//
export const resImages = (rest_id, cb) => {
  $.ajax({
    url: `http://localhost:3004/restaurants/${rest_id}/gallery`,
    method: "GET",
    success: resImages => {
      cb(rest_id, resImages);
    }
  });
};
