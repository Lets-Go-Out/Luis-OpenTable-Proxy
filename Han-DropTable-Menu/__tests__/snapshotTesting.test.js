import React from "react";
import Renderer from "react-test-renderer";
import Entries from "../src/component/entries.jsx";
import Menu from "../src/component/menu.jsx";
import { Provider } from "react-redux";
import Store from "../src/component/storage/store.js";
import fetch from "isomorphic-fetch"; //do not remove this!!! you need it for the stupid fetch that node doesn't have

describe("snapshotting", () => {
  test("snapshotting", () => {
    const snapshot1 = Renderer.create(
      <Provider store={Store}>
        <Menu />
      </Provider>
    ).toJSON();
    expect(snapshot1).toMatchSnapshot();
  });
});
