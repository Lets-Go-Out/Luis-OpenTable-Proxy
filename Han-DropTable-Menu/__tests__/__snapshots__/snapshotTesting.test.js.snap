// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`snapshotting snapshotting 1`] = `
<div
  className="MotherContainer"
>
  <div
    className="OtherComponents"
  />
  <div>
    <h2>
      Menu
    </h2>
    <div
      id="hidden"
    >
      <button
        onClick={[Function]}
      >
        Previous
      </button>
      <button
        onClick={[Function]}
      >
        Next
      </button>
    </div>
    <nav
      id="menuNav"
    >
      <button
        className="MenuNavButton2"
        onClick={[Function]}
      >
        Lunch
      </button>
      <button
        className="MenuNavButton"
        onClick={[Function]}
      >
        Dinner
      </button>
      <button
        className="MenuNavButton"
        onClick={[Function]}
      >
        Brunch
      </button>
      <button
        className="MenuNavButton"
        onClick={[Function]}
      >
        Happy Hour
      </button>
    </nav>
    <div
      className="container-1"
    >
      <div
        className="container"
      >
        <div
          className="leftContainer"
        />
        <div
          className="rightContainer"
        />
      </div>
      <div
        id="blocker"
      />
    </div>
    <div>
      <div
        style={
          Object {
            "borderBottom": "1px solid #d8d9db",
            "paddingBottom": "40px",
            "textAlign": "center",
          }
        }
      >
        <button
          id="viewFull"
          onClick={[Function]}
        >
          View All
        </button>
      </div>
    </div>
  </div>
  <div />
</div>
`;
